rosetta-source/src/test/java/org/isda/cdm/functions/FunctionInputCreationTest.java

```diff
@@ -385,25 +385,28 @@ void validatePartialTerminationVanillaSwapFuncInputJson() throws IOException {
     @Test
     void validatePartialTerminationEquitySwapFuncInputJson() throws IOException {
         // The tradeState input to partial termination is output from increase event
-        CreateBusinessEventInput increaseEquitySwapInput = getIncreaseEquitySwapFuncInputJson();
+        CreateBusinessEventInput increaseEquitySwapInput = getIncreaseEquitySwapExistingTradeLotFuncInputJson();
         BusinessEvent increaseOutput = runCreateBusinessEventFunc(increaseEquitySwapInput);
         TradeState increaseTradeState = increaseOutput.getAfter().get(0);
```

Comment:
This needs looking at.. 
- The original code gets the increase equity swap func input (which has 1 tradelot), and actually runs the function so a 2nd tradeLot is added. It then extracts the output (which has 2 tradeLots).. the output is used as the input to the partial termination example.
- The new code gets the increase equity swap with existing trade lot func input (which already has 2 tradeLots), so it's not really necessary to then run the function